yield_plugin/config/parameters.yaml

```diff
@@ -0,0 +1,9 @@
+acceleration_adjustment_factor: 4.0  # Adjustment factor for safe and comfortable acceleration/deceleration 
+collision_horizon: 10.0     # time horizon for collision detection in s
+min_obstacle_speed: 2.0     # Minimum speed for moving obstacle in m/s
+tpmin: 2.0                  # Minimum object avoidance planning time in s
+maximum_deceleration_value: 3.0 # Max deceleration value in m/s^2
+x_gap: 2.0                     # Minimum safety gap in m
+vehicle_length: 5.0            # Host vehicle length
+vehicle_width: 2.5             # Host vehicle width
+vehicle_height: 3.0            # Host vehicle height
```

Comment:
changed the node so the vehicle dimensions are pulled from config, this is just the default value. The max decel was actually max deceleration for yield, not the vehicle's limitation. I renamed it to avoid confusion.